autogen/main/dns.tf.tmpl

```diff
@@ -17,43 +17,15 @@
 {{ autogeneration_note }}
 
 /******************************************
-  Delete default kube-dns configmap
+  Manage kube-dns configmaps
  *****************************************/
-module "gcloud_delete_default_kube_dns_configmap" {
-  source  = "terraform-google-modules/gcloud/google//modules/kubectl-wrapper"
-  version = "~> 3.1"
-
-  enabled                     = (local.custom_kube_dns_config || local.upstream_nameservers_config) && !var.skip_provisioners
-  cluster_name                = google_container_cluster.primary.name
-  cluster_location            = google_container_cluster.primary.location
-  project_id                  = var.project_id
-  upgrade                     = var.gcloud_upgrade
-  impersonate_service_account = var.impersonate_service_account
```

Comment:
Lets remove these vars, IIRC they are not used anywhere else.

Reply:
👍